tests/test_unit/test_cdf_tk/test_commands/test_clean.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from pytest import MonkeyPatch
+
+from cognite_toolkit._cdf_tk import cdf_toml
+from cognite_toolkit._cdf_tk.cdf_toml import CDFToml
+from cognite_toolkit._cdf_tk.commands import CleanCommand
+from cognite_toolkit._cdf_tk.data_classes._config_yaml import BuildEnvironment
+from cognite_toolkit._cdf_tk.feature_flags import FeatureFlag
+
+
+@pytest.fixture
+def cdf_toml_v07_disabled(monkeypatch: MonkeyPatch):
+    """Fixture that modifies CDFToml singleton to disable v07 flag."""
+    # Clear the cache first to ensure fresh state
+    FeatureFlag.is_enabled.cache_clear()
+    my_cdf_toml = CDFToml.load(use_singleton=False)
+    my_cdf_toml.alpha_flags["v07"] = False
+    monkeypatch.setattr(cdf_toml, "_CDF_TOML", my_cdf_toml)
+    yield my_cdf_toml
+    # Cleanup: reset singleton and clear cache
+    monkeypatch.setattr(cdf_toml, "_CDF_TOML", None)
+    FeatureFlag.is_enabled.cache_clear()
+
+
+@pytest.fixture
+def cdf_toml_v07_enabled(monkeypatch: MonkeyPatch):
+    """Fixture that modifies CDFToml singleton to enable v07 flag."""
+    # Clear the cache first to ensure fresh state
+    FeatureFlag.is_enabled.cache_clear()
+    my_cdf_toml = CDFToml.load(use_singleton=False)
+    my_cdf_toml.alpha_flags["v07"] = True
+    monkeypatch.setattr(cdf_toml, "_CDF_TOML", my_cdf_toml)
```

Comment:
With my suggestion, then these fixtures are not necessary, and it becomes less to clean up when we do the v0.7 release. Instead, you can just assume the v0.7 behavior and test the new logic you have added. Additional benefit - less code :)